data/bash-completion/fwupdtool.in

```diff
@@ -64,22 +64,22 @@ _fwupdtool_opts=(
 _show_filters()
 {
 	local flags
-	flags="$(command @libexecdir@/fwupdtool get-device-flags 2>/dev/null)"
+	flags="$(command fwupdtool get-device-flags 2>/dev/null)"
```

Comment:
AFAICT this was the only thing being replaced by configuration data.  So perhaps `fwupdtool.in` should rename to `fwupdmgr` and then no more configuration data processing/replacing.